src/Logic/Storage.java

```diff
@@ -0,0 +1,50 @@
+package Logic;
+import java.io.*;
```

Comment:
Should import statements be listed explicitly instead of using import *? Noticed the same issue in several other classes too